packages/gafl-webapp-service/src/pages/order-complete/order-complete/__tests__/order-complete.spec.js

```diff
@@ -29,23 +29,29 @@ jest.mock('../../../../constants.js', () => ({
 jest.mock('../../../../processors/price-display.js')
 jest.mock('@defra-fish/business-rules-lib')
 
+const getSampleLicensee = ({
+  postalFulfilment = false,
+  preferredMethodOfConfirmation = 'phone',
+  preferredMethodOfReminder = 'phone'
+} = {}) => ({
+  postalFulfilment,
+  preferredMethodOfConfirmation,
+  preferredMethodOfReminder
+})
+
 const getSamplePermission = ({
   referenceNumber = 'AAA111',
   licenceType = LICENCE_TYPE['trout-and-coarse'],
   isLicenceForYou = true,
-  postalFulfilment = false,
-  preferredMethodOfConfirmation = 'phone',
-  preferredMethodOfReminder = 'phone'
+  licenceLength = '12M',
+  licensee = getSampleLicensee({ postalFulfilment: false, preferredMethodOfConfirmation: 'phone', preferredMethodOfReminder: 'phone' })
```

Comment:
Do we need to specify all the args for `getSampleLicensee()` here given that they match the default ones anyway?